docs/static_site/src/assets/js/globalSearch.js

```diff
@@ -0,0 +1,152 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/* Installation page display functions for install selector.
+   This utility allows direct links to specific install instructions.
+*/
+
+$(document).ready(function () {
+  const DEFAULT_CURRENT_VERSION = "1.6";
+  const VERSIONS = [
+    "master",
```

Comment:
Will need to update the [release manager runbook](https://cwiki.apache.org/confluence/display/MXNET/Release+Process), so they know that this needs updating... like the 1.7 release going on right now...

Reply:
Got it, I will move these to Jekyll config and update the runbook with the new place

Reply:
+1